src/datasources/asset/components/editors/calibration-forecast/CalibrationForecastEditor.tsx

```diff
@@ -70,7 +70,7 @@ export function CalibrationForecastEditor({ query, handleQueryChange, datasource
 
   function onParameterChange(ev: CustomEvent) {
     if (query.filter !== ev.detail.linq) {
-      query.filter = ev.detail.linq;
+      query.filter = filterXSSLINQExpression(ev.detail.linq);
```

Comment:
Why do we need to do this here if we're already sanitizing inside the query builder component?